### PR TITLE
Display import errors in UI

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

Relies on the api changing to work something like [this](https://gist.github.com/justinvdm/ac4044773cdaea4089ab).

File: client/src/views/git-importer/components/error-message/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      );
+
+    case 'page_source_not_exists':
+      return (
+        <div className="c-import-error-message">
+          <span className="o-text--primary o-text--label">{d.details.lang}</span> The source for page <span className="o-text--info">{d.details.page}</span> does not exist
+        </div>
+      );
+  }
+};
```
Comment (justinvdm) [inline client/src/views/git-importer/components/error-message/index.js]: We need better error messages here, but I think I first need to understand better what these errors mean.

**Comment by miltontony** [comment]
:+1: sweet!

**Comment by miltontony** [comment]
Merging this in order to continue reviewing #102

**Pull request merged**